scripts/modisSuite/Downloader.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python3
+# coding: utf8
+"""
+Auteur: Gabriel Desharnais
```

Comment:
I just changed the hardcoded \\ into the os file extenstions. I can add inline comments where I do this so we can push it back. They are all contained to Downloader.py